x-pack/plugins/security_solution/public/management/components/endpoint_responder/lib/console_commands_definition.ts

```diff
@@ -575,6 +553,24 @@ const adjustCommandsForSentinelOne = ({
   const isKillProcessEnabled = featureFlags.responseActionsSentinelOneKillProcessEnabled;
 
   return commandList.map((command) => {
+    // Kill-Process: adjust command to accept only `processName
+    if (command.name === 'kill-process') {
```

Comment:
Maybe we could turn it into an enum, the `kill-process`?

Reply:
Not a fan of Enum... They are hard to work with and often require type casting when needing to use a string.